questionReply/questionReply-library/src/main/java/org/silverpeas/components/questionreply/service/SilverpeasQuestionManager.java

```diff
@@ -73,12 +74,11 @@
 
   SilverpeasQuestionManager() {
     try {
-      questionDao = SilverpeasBeanDAOFactory.<Question>getDAO(Question.class.getName());
-      replyDao = SilverpeasBeanDAOFactory.<Reply>getDAO(Reply.class.getName());
-      recipientDao = SilverpeasBeanDAOFactory.<Recipient>getDAO(Recipient.class.getName());
+      questionDao = SilverpeasBeanDAOFactory.getDAO(Question.class.getName());
+      replyDao = SilverpeasBeanDAOFactory.getDAO(Reply.class.getName());
+      recipientDao = SilverpeasBeanDAOFactory.getDAO(Recipient.class.getName());
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Remove this use of "SilverpeasBeanDAOFactory"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)